dev/provider_packages/prepare_provider_packages.py

```diff
@@ -125,21 +125,20 @@ def cli():
     ...
 
 
-@cli.resultcallback()
-def process_result(result):
-    # This is special case - when the command executed returns false, it means that we are skipping
-    # the package
-    if result is False:
-        raise click.exceptions.Exit(64)
-    return result
-
-
 option_git_update = click.option(
     '--git-update/--no-git-update',
     default=True,
     is_flag=True,
     help=f"If the git remote {HTTPS_REMOTE} already exists, don't try to update it",
 )
+
+option_interactive = click.option(
+    '--interactive/--no-interactive',
```

Comment:
```suggestion
    '--interactive/--non-interactive',
```

Worth changing `--no-interactive` to `non-interactive` everywhere

Reply:
I am afraid this is not how click works :( https://click.palletsprojects.com/en/7.x/options/#boolean-flags

Reply:
Ah yeah Indeed.